cmd/serve.go

```diff
@@ -305,7 +305,14 @@ func StartServer(logger *log.Zap, cfg *config.Frontier) error {
 
 	go server.ServeUI(ctx, logger, cfg.UI, cfg.App)
 
-	// serving server
+	// start connect server
+	go func() {
+		if err := server.ServeConnect(ctx, logger, cfg.App.Connect, deps); err != nil {
```

Comment:
it does not seem like we are performing a graceful shutdown here. Can you please check on this?

Reply:
Yes, great catch. Thanks. Added the shutdown part.